rfc/2/index.md

```diff
@@ -260,7 +256,8 @@ File hierarchy of one multi-scale OME-Zarr image `456.zarr`:
   "zarr_format": 3,
   "node_type": "group",
   "attributes": {
-    "https://ngff.openmicroscopy.org/0.6": {
+    "ome": {
+      "version": "0.5",
       "multiscales": [
         {
           "coordinateSystems": [
```

Comment:
The `coordinateSystems` here and the `"input": "/1", "output": "root"` attributes below seem to come from the previous v0.5 proposal but shouldn't be included in this one. Also the `"discrete": true` from `axes`?

Reply:
Yes, that is true. I had adapted that to the coordinate transforms proposal.